xclim/core/units.py

```diff
@@ -42,6 +41,7 @@
 
 
 units = pint.UnitRegistry(autoconvert_offset_to_baseunit=True, on_redefinition="ignore")
+Convertible = TypeVar("Convertible", str, xr.DataArray, units.Unit, units.Quantity)
```

Comment:
Neat! Can you briefly explain this syntax? How will this render in the docs?

Reply:
The annotation shows `"Convertible"`, so it seems equivalent to 
```
Convertible = str | xr.DataArray | units.Unit | units.Quantity 
```